chainercv/transforms/image/ten_crop.py

```diff
@@ -0,0 +1,37 @@
+import numpy
+
+
+def ten_crop(img, output_shape):
+    """Crop 10 regions from an array.
+
+    This method crops 10 regions. All regions will be in shape
+    ``output_shape``. These regions consists of 1 center crop and 4 corner
+    crops and horizontal flips of them.
```

Comment:
Also, could you explain the order in which 10 crops are stacked?
e.g. First element is center crop, ....